sklearn/decomposition/pca.py

```diff
@@ -353,6 +353,15 @@ def fit_transform(self, X, y=None):
         -------
         X_new : array-like, shape (n_samples, n_components)
 
+
+        ..note:: This method changes a c-order array to an f-order array.
```

Comment:
Use:

```rst
Notes
-----
```

This function will return a `f`-ordered array if `x` is c-order or f-ordered. (`x` does not have to be `c`-ordered)

Reply:
Thanks @thomasjpfan , I've made the changes